tiledb/cloud/config.py

```diff
@@ -91,7 +93,10 @@ def load_configuration(config_path):
                 verify_ssl = config_obj["verify_ssl"]
 
     if (token is None or token == "") and (username is None or username == ""):
-        return "You must first login before you can run commands"
+        warnings.warn(
+            "You must first login before you can run commands. Please run tiledb.cloud.login."
+        )
+        logged_in = False
```

Comment:
Any reason not to `return False` here? (in which case we can also lose the `logged_in` variable and `return True` at the end of the function)

Reply:
Well some public assets and functions work without being logged in, but we've got to configure the host for it to work. There is probably a cleaner way to do this, or a better message to have but I didn't want the warning to get too long.

`tiledb.cloud.info("tiledb://TileDB-Inc/quickstart_sparse")`